homeassistant/components/fritz/services.py

```diff
@@ -13,18 +13,25 @@
     SERVICE_CLEANUP,
     SERVICE_REBOOT,
     SERVICE_RECONNECT,
+    SERVICE_SCHEMA_SET_GUEST_WIFI_PW,
+    SERVICE_SET_GUEST_WIFI_PW,
 )
 
 _LOGGER = logging.getLogger(__name__)
 
 
-SERVICE_LIST = [SERVICE_CLEANUP, SERVICE_REBOOT, SERVICE_RECONNECT]
+SERVICE_LIST = [
```

Comment:
```suggestion
SERVICE_LIST: list[tuple[str, vol.Schema | None]] = [
```
would add some typing